.travis.yml

```diff
@@ -8,6 +8,7 @@ services:
 php:
   - 7.2
   - 7.3
+  - 7.4snapshot
```

Comment:
Well, *7.4* is not yet integrated into travis. Try with snapshot, or maybe we should wait...